Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AuthMethodManager.java

```diff
@@ -76,6 +77,17 @@ public Azure getAzureClient(String sid) throws IOException {
         return azure;
     }
 
+    public AppPlatformManager getAzureSpringCloudClient(String sid) throws IOException {
+        if (getAzureManager() == null) {
+            throw new IOException(CANNOT_GET_AZURE_MANAGER);
+        }
+        AppPlatformManager client = getAzureManager().getAzureSpringCloudClient(sid);
+        if (client == null) {
+            throw new IOException(String.format(CANNOT_GET_AZURE_BY_SID, sid));
```

Comment:
throw the exception in `getAzureSpringCloudClient` method?

Reply:
Done